src/CONST.ts

```diff
@@ -1201,6 +1201,8 @@ const CONST = {
         NOTE: 'n',
     },
 
+    IMAGE_HIGH_RESOLUTION_THRESHOLD: 5000,
```

Comment:
How did we end of choosing 5000 as the threshold?

Reply:
I use [this suggestion](https://github.com/Expensify/App/issues/38180#issuecomment-2191136178) as reference, should we increase it?

cc: @Julesssss

Reply:
Ah, to be honest that was a bit of a guess. I'm open to suggestions here

Reply:
@wildan-m lets start with 7000px and match Slack.